sky/cli.py

```diff
@@ -3164,18 +3189,40 @@ def _output():
             else:
                 name, quantity = accelerator_str, None
 
+        print_section_titles = False
+        if kubernetes_is_enabled and (cloud is None or
+                                      cloud_is_kubernetes) and not show_all:
```

Comment:
nit: for better format
```suggestion
        if (kubernetes_is_enabled and (cloud is None or
                                      cloud_is_kubernetes) and not show_all):
```